consul/resource_consul_node_test.go

```diff
@@ -23,6 +23,25 @@ func TestAccConsulNode_basic(t *testing.T) {
 					testAccCheckConsulNodeValue("consul_node.foo", "name", "foo"),
 				),
 			},
+			resource.TestStep{
+				PreConfig: func() {
```

Comment:
Can you create a helper function like you did for `testAccRemoveConsulIntention`, it will avoid deeply nesting tests.